build/linux/installer/conf/fluent-bit-geneva-logs_tenant.conf

```diff
@@ -18,6 +18,19 @@
     Exclude_Path ${AZMON_CLUSTER_LOG_TAIL_EXCLUDE_PATH}
     ${TAIL_THREADED}
 
+#${KubernetesFilterEnabled}[FILTER]
```

Comment:
This should be moved to fluent-bit-geneva.conf after multi-line and this similar to multi-line filter . We only need one filter all up geneva.